Source/PeleLMTemporals.cpp

```diff
@@ -2,19 +2,37 @@
 
 using namespace amrex;
 
-void PeleLM::initTemporals()
+void PeleLM::initTemporals(const PeleLM::TimeStamp &a_time)
 {
    if (!m_do_temporals
        && !(m_nstep % m_temp_int == 0)) return;
 
    // Reset mass fluxes integrals on domain boundaries
    if (m_do_massBalance && !m_incompressible) {
-      m_massOld = MFSum(GetVecOfConstPtrs(getDensityVect(AmrOldTime)),0);
+      m_massOld = MFSum(GetVecOfConstPtrs(getDensityVect(a_time)),0);
       for (int idim = 0; idim <  AMREX_SPACEDIM; idim++) {
          m_domainMassFlux[2*idim] = 0.0;
          m_domainMassFlux[2*idim+1] = 0.0;
       }
    }
+   m_RhoHOld = MFSum(GetVecOfConstPtrs(getRhoHVect(a_time)),0);
+   for (int idim = 0; idim < AMREX_SPACEDIM; idim++) {
+      m_domainRhoHFlux[2*idim] = 0.0;
+      m_domainRhoHFlux[2*idim+1] = 0.0;
+   }
+
+   for (int n = 0; n < NUM_SPECIES; n++){
+      m_RhoYOld[n] = MFSum(GetVecOfConstPtrs(getSpeciesVect(a_time)),n);
+      for (int idim = 0; idim < AMREX_SPACEDIM; idim++) {
+         m_domainRhoYFlux[2*n*AMREX_SPACEDIM+2*idim] = 0.0;
+         m_domainRhoYFlux[1+2*n*AMREX_SPACEDIM+2*idim] = 0.0;
+      }
+   }
+   for (int idim = 0; idim < AMREX_SPACEDIM; idim++) {
+      m_domainUmacFlux[2*idim] = 0.0;
+      m_domainUmacFlux[2*idim+1] = 0.0;
+   }
```

Comment:
This should be done regardless of the `m_do_temporal` status right ?

Reply:
pulling out and placing in PeleLMAdvance